example/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java

```diff
@@ -12,8 +12,8 @@ public static void registerWith(PluginRegistry registry) {
     if (alreadyRegisteredWith(registry)) {
       return;
     }
-    CloudFirestorePlugin.registerWith(registry.registrarFor("io.flutter.plugins.firebase.cloudfirestore.CloudFirestorePlugin"));
```

Comment:
I remember seeing instructions on how to port this part like you did, but I can't find it in https://firebase.flutter.dev/docs/migration/ anymore. Do you know where those migration instructions went?

Reply:
hmm.. I found that we should gitignore theses Generated***.
See: https://github.com/FirebaseExtended/flutterfire/blob/a0846414ee36133bacb24b96c232b03c99571862/.gitignore#L37-L41

Reply:
created issue https://github.com/atn832/cloud_firestore_mocks/issues/128